src/components/LinkSDK/index.js

```diff
@@ -41,14 +40,13 @@ const LinkSDK = forwardRef((props, ref) => {
             }
             `;
 
-      SDK.current.injectJavaScript(call);
+      setInjectedJavascript(call);
+      setIsOpen(true);
```

Comment:
Can we extract this to a reusable function that takes the string?

Also we should rename the string to something more descriptive of it's purpose.